File: JavaScript/8js 2/oddCount.js
/*
|--------------------------------------------------------------------------
| Count Odd Numbers below n
|--------------------------------------------------------------------------
|
*/

function oddCount(n) {
   return Math.floor(n / 2);
}


function oddCount(n) {
   return parseInt(n / 2);
}


function oddCount(n) {
   return Math.trunc(n / 2);
}


function oddCount(n) {
   return (n - n % 2) / 2
}


console.log(oddCount(15), 7, "Oops! Wrong.");
console.log(oddCount(15023), 7511, "Oops! Wrong.");

File: Python/8py 2/odd_count.py
"""
|--------------------------------------------------------------------------
| Count Odd Numbers below n
|--------------------------------------------------------------------------
|
"""

from math import floor
def odd_count(n):
   return floor(n/2)


def oddCount(n):
   return n // 2


def odd_count(n):
   return len(range(1, n, 2))


def oddCount(n):
   return n >> 1


print(odd_count(15),7)
print(odd_count(15023),7511)